### ISSUE TITLE
[Feature] Simpler UI window

### ISSUE THREAD

**Issue by rwjack** [nlpsuge/gnome-shell-extension-another-window-session-manager#70]
I've noticed that I struggle to find what I want every time when I open the little window:

![image](https://user-images.githubusercontent.com/59068073/220122342-21131bcf-bbc6-43e6-a743-a8165890b821.png)

The 3 operation buttons are awfully similar and confusing (save|restore|move), so I have to stop and think which one I actually want to press.

Often times I'm just restoring windows from a saved session, and second to that I sometimes move windows to their respective workspaces, and I'm sure other users do too.

Also, why is there a save date next to each session? I don't think that's relevant information and it's just confusing. Eg. I know it's a "work" session, I just want to restore the windows, no other info needed. Though I did notice there is no visual confirmation when saving a session other than the timestamp updating.
A solution then would be to show the timestamps on debug mode or just don't show them at all - If someone needs to check the session save time, that's basically for debugging purposes and should be saved in the .desktop file for further investigation.

Next, the search bar? How many sessions do users usually have, 50?  Unlikely, I think that's also just a waste of space.

Same for the recently closed session, but that might just be me not using it at all.

To recap, this UX feature request would be:
- make operation icons more distinct and bigger
- remove session save timestamps (or show only on debug mode)
- remove search bar
- implement a setting toggle to show/hide "Recently Closed Session"

**Comment by nlpsuge** [comment]
> * make operation icons more distinct and bigger

How about adding some options to the Preferences under `Appearance` tab?
* `Operation icon size`
The current icon size is 14, which is too small indeed. I can give it a bigger default size (maybe 18 or even bigger, I have to test it myself on my machine), but anyway you can set the icon to a comfortable size in you own computer via this option. This setting affects that three operation icons (save|restore|move) on the popup window. 

* `Font size`
This setting can set the font size of the popup window.
The default font size is also too small to me. And if the operation icon size becomes bigger, the small font might be weird. 


> Though I did notice there is no visual confirmation when saving a session other than the timestamp updating.

I can send a notification to let the user know, once the session is saved. 

> * remove session save timestamps (or show only on debug mode)

OK, it will show only on debug mode. Thanks for the suggestion.

> * remove search bar [and] implement a setting toggle to show/hide "Recently Closed Session"

I can add a setting in the `Appearance`. The search bar will be hidden by default. I personally sometimes probably need to search the saved session list.

I'm working [on this branch](https://github.com/nlpsuge/gnome-shell-extension-another-window-session-manager/tree/feature-close-active-app) to add three tabs to the popup window , looks like this:
![image](https://user-images.githubusercontent.com/2271720/220671604-76060b61-20cb-4651-9a66-e9b41ba8d8f7.png)

The search bar should be useful in the Running Apps/Windows, users can search a window by typing title name and close them using AWSM. And `Recently Closed Session` will be removed while the `Recently Closed` tab is finished. It's a mistake to add the `Recently Closed Session`.




Thanks again for the suggestions.

**Comment by rwjack** [comment]
Your alternatives are even better than my suggestions.

Glad we're resonating on the same frequency here!
